config/config.go

```diff
@@ -477,7 +479,7 @@ func (cfg *Manager) SaveJSON(path string) error {
 
 // ToJSON provides a JSON representation of the configuration by
 // generating JSON for all componenents registered.
-func (cfg *Manager) ToJSON() ([]byte, error) {
+func (cfg *Manager) ToJSON(display bool) ([]byte, error) {
```

Comment:
I'm not happy about this. What is `display`? Anyone reading the code will find that ToJSON() is taking true, or false. So there's an extra step about figuring out what this true or false means.